test/regression/2015-08-05-invalid-fork.c

```diff
@@ -0,0 +1,24 @@
+/* Reported by @kren1 in #262 
+   The test makes sure that the string "Should be printed once" 
+   is printed a single time. 
+*/
+
+// RUN: %llvmgcc %s -emit-llvm -g -O0 -c -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: %klee --output-dir=%t.klee-out %t.bc >%t1.log
+// RUN: grep "Should be printed once"  %t1.log
```

Comment:
@delcypher , I think we should use whatever is most comfortable for people.  And grep is certainly a reasonable dependency.   In any case, how do you check that a line occurs exactly once with FileCheck?